app/views/posts/show.html.erb

```diff
@@ -16,8 +16,18 @@
 
 <div class="border border-dark border-3 mx-auto w-100 p-3 mb-3">
 <% @post.most_recent_comments.each do |comment| %>
-<p> <%= "#{comment.author.name}: #{comment.text}" %></p>
+ <%= "#{comment.author.name}: #{comment.text}" %>
 <% end %>
 </div>
+
+<div>
+  <h2> Add Comments</h2>
+    <%= form_for [@post, Comment.new], url: "/comments/#{@post.id}" do |form| %>
+     <div>
+        <%= form.text_area :text, placeholder: "write new comment..", class:"form-control" %>
+        </div>
+        <%= form.submit "Add Comment", class: "btn btn-primary mb-2" %>
+    <% end %>
+</div>
 </section>
 ```
```

Comment:
- [ ] Kindly remove these three backticks. I think it is a mistake.

- [ ] **[OPTIONAL]** You have done well adding the like create method in the controller but I think it is more appropriate to also add the like button on the post show page as well.

  ![image](https://github.com/brhanuhailu/blog-app/assets/81468675/bda8d093-a09f-4c05-abe9-0d1ed8f0aa6a)